### PR TITLE
Simplified/revised CoC

### PR DESCRIPTION
**Pull request by ebullient**
- Contributor Covenant v1.4
- Emphasize that disagreement is ok/expected

### CODE REVIEW

File: policies/code-of-conduct.md [hunk 1]
```diff
@@ -66,32 +55,41 @@ If you're interested in an introduction to diversity, inclusion, and unconscious
...
 
-Confidentiality is of utmost importance. We also understand that the person reporting the issue (**reporter**) may not be the individual directly affected or harmed (**target**). In such cases, communication will primarily be with the reporter unless the target explicitly gives permission. Personal details of the reporter (and the target if they differ) will not be disclosed without permission.
+For issues that escalate beyond direct resolution or involve behavior contrary to our Code of Conduct, the CF provides clear avenues for reporting:
 
-- The CF *prohibits retaliation* against individuals who report concerns in good faith.
+- **Project-Specific Reporting**: Each CF project may implement its own CoC enforcement processes suitable to its size and scope. These processes and contact methods should be detailed in the project's `CODE_OF_CONDUCT.md`.
+- **Foundation-Wide Reporting**: For concerns within CF spaces not covered by a specific project's `CODE_OF_CONDUCT.md`, or for issues that require escalation beyond the project level, reports should be directed to the [report mailing list][CONTACTS.yaml]. The CFC will oversee these reports.
```
Comment (kenfinnigan) [inline policies/code-of-conduct.md:76]: Should this be clearer to indicate projects without a specific CoC are covered by the CF one? "CF spaces", to me, left room for uncertainty.

One possible alternative is "For concerns within the CF, projects not covered by their own CoC, or for issues that require escalation beyond the project level"?

**Review by kenfinnigan — APPROVED** [review]

File: policies/code-of-conduct.md (hunk 1 above)
Comment (ebullient) [reply policies/code-of-conduct.md:76]: sure. suggest? I was smashing a lot of text..

Comment (ebullient) [reply policies/code-of-conduct.md]: Took a stab at it by just removing the extra words.. WDYT?

Comment (kenfinnigan) [reply policies/code-of-conduct.md]: I suggested:
```
For concerns within the CF, projects not covered by their own CoC, or for issues that require escalation beyond the project level, reports should be directed to the [report mailing list][CONTACTS.yaml]. The CFC will oversee these reports.
```

Comment (ebullient) [reply policies/code-of-conduct.md]: oh! yes. you didn't use the +- thing so I could just accept it. Sorry. I read too fast. ;)

Comment (kenfinnigan) [reply policies/code-of-conduct.md]: What you have now is good!

**Review by gavinking — APPROVED** [review]

**Comment by gavinking** [comment]
I wonder if there's anything we could do/say to encourage the use of real names in our communities. [This is, like, the #1 best way to improve behavior online.]

**Pull request merged**